.github/workflows/workflow_test.yaml

```diff
@@ -75,4 +76,4 @@ jobs:
           [ '${{ needs.dockerfile.outputs.docker-lint-outcome }}' = 'success' ] || (echo docker-lint failed && false)
           [ '${{ needs.integration.result }}' = 'success' ] || (echo integration failed && false)
           [ '${{ needs.integration-rock.result }}' = 'success' ] || (echo integration-rock failed && false)
-          [ '${{ needs.publish.result }}' = 'success' ] || (echo publish failed && false)
+          [ '${{ needs.publish.result }}' != 'failure' ] || (echo integration-rock failed && false)
```

Comment:
Should this be `publish` rather than `integration-rock`?

Reply:
Yes, indeed. Updated, thanks!